### DISCUSSION TITLE
I've just booted up chromebook_snow-armv7l-bookworm.img successfully, but it's asking for a password

### DISCUSSION THREAD

**Discussion by Martyn575** [velvet-os/imagebuilder#250] [Q&A] [answered]
Hello, thanks for building these images. 

I've just booted up chromebook_snow-armv7l-bookworm.img successfully - thank you very much! 

I'm having serious trouble however logging in. I've been told the password is "changeme", but this password doesn't work. 

I've gone to the terminal to login as root with this, but the root login is disabled. Do you know the username also? I just need to login, and i'm completely locked out. 

Kind regards!

**Comment by thenameisluk** [comment, marked as answer]
@Martyn575 
root account is disabled by default

if you want to access it just login as linux/changeme and run
```
sudo passwd root
```
to set the password for root

**Comment by Martyn575** [comment]
I'm so sorry, what happened was that i couldn't remember whether it booted from USB or booted from SD card, so i left the SD card with an old version of linux in there (that had a different login), and it booted off that rather than the new version in the USB drive. The password is indeed linux / changeme! Feel free to delete this discussion if you like! Thank you for your reply. Also I did get the system running linux perfectly last night - nice work :)
